source/chapters/1/intro.md

```diff
@@ -91,37 +101,50 @@ This is the basic folder structure that an application, such as the one you've j
     └── main.js // this file contains: import /imports/startup/server
 </pre>
 
+This is not the structure of the application you have just created ! 
+You need to create a new structure for the application, in order for your project to be better organized, and, as a 
+result, have a somewhat easier life. To do this, run these commands in your terminal :
+
 ```
-// to create it quickly, run the commands (they should work just fine on Mac/Linux)
 mkdir -p client server imports/ui imports/startup/client imports/startup/server imports/api
 echo "import '/imports/startup/client';" > client/main.js
 echo "import '/imports/startup/server';" > server/main.js
 ```
-
+![Windows](../images/windowsSmall.png "Windows icon")
+On windows, the script syntax is somewhat different. To achieve the same results on Windows as you would with the 
+previous script on Linux, run these command into your CMD or PowerShell:
+```
+echo ./imports, ./imports/api, ./imports/startup , ./imports/ui, ./startup/client, ./startup/server | % { mkdir "$_" }
+echo "import '/imports/startup/client';" > client/main.js
+echo "import '/imports/startup/server';" > server/main.js
+```
 
 Everything in "/client" and "/server" is automatically loaded, as it is explained [here.](https://guide.meteor.com/structure.html#load-order)
 
 That's all great ! However, we need more control over our application, and in order to gain that control,
 we will refrain from using auto-loading in our projects.
 
-Everything in "/imports" must be explicitly imported so Meteor can "gain knowledge" about it.
-This gives us the ability to build modular applications.
+This compiles all the modules when we type **"meteor run"** into the command line, but does ot load all of them.  
+
+Because of this, everything in "/imports" must be explicitly imported so Meteor can "gain knowledge" about it.
+This also gives us the ability to build modular applications.
 [Read more about this here.](https://danmartensen.svbtle.com/build-better-apps-with-es6-modules)
 
 
 ## Importing from NPM
 
-For now, we won't get into a lot of details, but the basic idea is that with Meteor you have access to the full NPM ecosystem, allowing you to import
-modules from http://www.npmjs.com
+For now, we won't get into a lot of details, but the basic idea is that with Meteor you have access to the
+ full NPM ecosystem, allowing you to import modules from http://www.npmjs.com
 As an idea, to get you to realise just how many resources you have at your disposal, the NPM system contained roughly 445 797 packages on May 1st of 2017.
 
+"moment" is a library you will use when working with dates in your projects.
+You install it by running the following command into your terminal:
 ```
-// "moment" is a library you will most likely use when working with dates
-// This is how you install it:
 meteor npm install --save moment
```

Comment:
Specify the exact location where they need to be. If they open their terminal and they are on a random place, it won't work. This should be like a reminder and from here on, this thing won't be specified anymore

Reply:
Explained how this works, with an example. :)